### PR TITLE
Add logging of fts and cartesian wrenches on left and right arm and right upper leg when using iCubGazeboV3

### PR DESCRIPTION
**Pull request by isorrentino**

### CODE REVIEW

File: devices/YarpRobotLoggerDevice/app/robots/iCubGazeboV3/interface/right_upper_leg_wrench_client.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+This software may be modified and distributed under the terms of the
+GNU Lesser General Public License v2.1 or any later version. -->
+<?xml version="1.0" encoding="UTF-8" ?>
+<device xmlns:xi="http://www.w3.org/2001/XInclude" name="right_upper_leg_wrench_client" type="genericSensorClient">
+    <param name="remote">/wholeBodyDynamics/right_upper_leg/cartesianEndEffectorWrench:o</param>
+    <param name="local">/yarp_robot_logger/right_upper_leg_wrench_client</param>
+</device>
+
+
+
```
Comment (GiulioRomualdi) [inline devices/YarpRobotLoggerDevice/app/robots/iCubGazeboV3/interface/right_upper_leg_wrench_client.xml:11]:
```suggestion



```

**Review by GiulioRomualdi — COMMENTED** [review]
Minor comment

**Review by prashanthr05 — APPROVED** [review]
@isorrentino maybe this would be a good time to reorganize the multiple FT sensors files into a single file for better maintenance?

I was also wondering if we should start moving towards Multiple Analog Sensors for the FTs since analog sensors are getting deprecated?

**Comment by isorrentino** [comment]
> @isorrentino maybe this would be a good time to reorganize the multiple FT sensors files into a single file for better maintenance?

Maybe I can start adding this modification to this PR.



> I was also wondering if we should start moving towards Multiple Analog Sensors for the FTs since analog sensors are getting deprecated?

and leave this for a new PR. @prashanthr05 what do you think?

**Comment by prashanthr05** [comment]
Yep, that would be neater.

**Comment by isorrentino** [comment]
> @isorrentino maybe this would be a good time to reorganize the multiple FT sensors files into a single file for better maintenance?

Hi @prashanthr05,

I added the modifications in the latest commit.

**Comment by isorrentino** [comment]
Friendly ping @prashanthr05 @GiulioRomualdi

**Comment by GiulioRomualdi** [comment]
@isorrentino can you please update the changelog adding this: https://github.com/ami-iit/bipedal-locomotion-framework/pull/525#issuecomment-1096700239

**Comment by GiulioRomualdi** [comment]
I will merge the PR after this https://github.com/ami-iit/bipedal-locomotion-framework/pull/525#issuecomment-1110834405 even if the CI is failing. Indeed this modification do not affect the compilation issue we are experiencing in te CI

**Comment by isorrentino** [comment]
Done @GiulioRomualdi

**Pull request merged**